test/common/EnvVars.cpp

```diff
@@ -195,7 +196,8 @@ namespace RcclUnitTesting
         std::make_tuple("UT_MAX_RANKS_PER_GPU", maxRanksPerGpu, "Maximum number of ranks using the same GPU"),
         std::make_tuple("UT_PRINT_VALUES"     , printValues   , "Print array values (-1 for all)"),
         std::make_tuple("UT_SHOW_TIMING"      , showTiming    , "Show timing table"),
-        std::make_tuple("UT_INTERACTIVE"      , useInteractive, "Run in interactive mode")
+        std::make_tuple("UT_INTERACTIVE"      , useInteractive, "Run in interactive mode"),
+        std::make_tuple("UT_TIMEOUT"          , timeout       , "Timeout limit for collective calls")
```

Comment:
Specify what units this timeout limit is in.  You have in comments that it is in microseconds, but haven't explained this to users.